app/helpers/application_helper.rb

```diff
@@ -1,3 +1,19 @@
 # View helpers that should be available application-wide.
 module ApplicationHelper
+  def image_asset_paths
+    asset_paths = {}
+    if Rails.env.development?
+      Rails.application.assets.each_file do |f|
```

Comment:
@chrisdolendo had to learn this myself but `Rails.application.assets` is something that is available during development, it's basically a list of all your assets (which aren't "precompiled" in development") however in prod it works differently, you have to refer to the `assets_manifest` which is like a record of all the assets that have been precompiled. In both cases the `image_asset_paths` result is the same, which is an object with keys being the name of the file and values being the hashed asset path, so we can reference this in JS:

```javascript
var IMAGE_ASSET_PATHS = {
 "arrow-down.png": "/assets/arrow-down-a443598bdf41cfdc56f7c3bab8ccf69425e5e4ebbaa47dd4a217ad05201f2cbb.png"
}
```